### PR TITLE
List inherited constructors on the class synopsis pages

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

**Comment by kocsismate** [comment]
@cmb69 Can this be merged instead of my "default" constructor related additions?

File: reference/intl/intlcodepointbreakiterator.xml [hunk 1]
```diff
@@ -48,6 +48,9 @@
     </xi:include>
 
     <classsynopsisinfo role="comment">&InheritedMethods;</classsynopsisinfo>
+    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.intlbreakiterator')/db:refentry/db:refsect1[@role='description']/descendant::db:constructorsynopsis[@role='IntlBreakIterator'])">
+     <xi:fallback/>
+    </xi:include>
```
Comment (cmb69) [inline reference/intl/intlcodepointbreakiterator.xml:53]: The "inherited" constructor is private, and as such is not really inherited. It is documented to convey that the class may not be instantiated, but I think we should document the class as being abstract instead (and document that it is not actually abstract), and, of course,  actually make it abstract.

**Review by cmb69 — COMMENTED** [review]
This is generally fine. Thank you!

We should figure out what to do regarding private constructors, though (see comment below).

File: reference/intl/intlcodepointbreakiterator.xml (hunk 1 above)
Comment (kocsismate) [reply reference/intl/intlcodepointbreakiterator.xml:53]: Nice catch! In my opinion, simply skipping these private constructors from the inherited method section is the way to go. Would you be ok with this approach?

Comment (cmb69) [reply reference/intl/intlcodepointbreakiterator.xml:53]: Yeah, sure. Would we need to remove/change the roles for that, or do you have another idea?

Comment (kocsismate) [reply reference/intl/intlcodepointbreakiterator.xml:53]: No, we don't need to change the markup, gen_stub can handle it :) See https://github.com/php/php-src/pull/10127/commits/b122959b5b0bf0a9d96a475b7692422dcdb0edae

Comment (cmb69) [reply reference/intl/intlcodepointbreakiterator.xml]: Great!

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**